Tests/CertificationTests.cs

```diff
@@ -0,0 +1,190 @@
+﻿using AventStack.ExtentReports;
+using AventStack.ExtentReports.Reporter;
+using Mars_CompetitionTask_US2.Pages;
+using Mars_CompetitionTask_US2.Utils;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using static Mars_CompetitionTask_US2.Utils.CommonMethods;
+
+namespace Mars_CompetitionTask_US2.Tests
+{
+    [TestFixture]
+    [Parallelizable]
+    public class CertificationTests : CommonMethods
+    {
+        [SetUp]
+        public void Setup()
+        {
+            
+            Login loginObj = new Login();
+            loginObj.LoginInToApplication();
+
+        }
+
+        [Test, Order(1)]
+        public void TC1_UserAddsCertification()
+        {
+            Certification certificationObj = new Certification();
+            test = extent.CreateTest(TestContext.CurrentContext.Test.Name);
+            certificationObj.CertificationTab();
+            // Parsing the Certification Json test data
+            JObject obj = GetTestDataJsonObjectFromFile("Certification");
+
+            //Fetching the particular array in Certification json file for reading the test data 
+            List<CertificationDetails> certifications = obj["Certification0"].ToObject<List<CertificationDetails>>();
+
+            for (int i = 0; i < certifications.Count; i++)
+            {
+                certificationObj.AddCertificate(certifications[i].certificate.ToString(), certifications[i].certifiedFrom.ToString(), certifications[i].year.ToString());
+                test.Log(Status.Pass, "Success");
+
+            }
+
+            if (certificationObj.VerifyAddedCertification())
+            {
+                Assert.Pass("Certification has been added");
+            }
+            else
+            {
+                Assert.Fail("Failed to add certification");
+            }
+
+
+        }
+
+
+
+        [Test, Order(2)]
+        public void TC2_UserEditsCertification()
+        {
+            Certification certificationObj = new Certification();
+            test = extent.CreateTest(TestContext.CurrentContext.Test.Name);
+            certificationObj.CertificationTab();
```

Comment:
so i see you thought about clean up state for education what about certification 

example before adding language you need to make sure there a no existing languages as this could result in false positive test pass
you will need to think about this for all the scenarios

the state is say that you have a test that is adding a language english, the second time you run this test you should delete any existing language so that the state is reset, this will guarantee that your test does not produce false negatives and passes always
you can assume that test execution order is not always guaranteed
imagine if you run the tests  in parallel mode the all tests will run at the same time
so you need to also make sure you do not wipe out other tests data
hence why when you remove data the best thing to do is delete only data you added for the test to be correct
e.g test 1 added english and french as language test 2 added hindi then when you reset data you already know what language you are adding for each test
there for on reset of the two test you should delete english and french for test 1 and hindi for test 2

So to sum up you should delete all data if there is any before the start of any test and then after each test you should delete the data that a particular test is adding!
This is ensure that upon staring the test pack you have reset state and upon finishing each test deletes their data that they added!

also say you are updating a language then you must added language then update it do not assume the language is there!



so think about first clean up before any test runs (you delete all data around this feature e.g  before any language test run you delete all languages). Then after each test delete only data that the test has added! (test1 added english and test2 to added french then test1 should only delete english and test2 should delete french only) 

also what about the cleanup after test?